### DISCUSSION TITLE
Contributing in another programming language?

### DISCUSSION THREAD

**Discussion by Curovearth** [ossamamehmood/Hacktoberfest#1501] [Q&A] [answered]
Hey, 
Finally, the awaited Hacktoberfest has arrived and I am really excited to contribute. Just a question that given you are the maintainer of the repo, is it possible for me to contribute to the repo in another programming language like Wolfram, Brainfuck or something else?

Thanks

**Comment by Thre4dripper** [comment, marked as answer]
Hacktoberfest is all about contribution to the open source. So it doesn't matter in what language you do contributions, just search any repo of your language choice who is participating in this event and you can contribute to it.
